clients/bigquery/converters/converters_test.go

```diff
@@ -81,7 +85,7 @@ func TestStringConverter_Convert(t *testing.T) {
 			// Time
 			val, err := NewStringConverter(typing.Time).Convert(time.Date(2021, 1, 1, 9, 10, 11, 123_456_789, time.UTC))
 			assert.NoError(t, err)
-			assert.Equal(t, "09:10:11.123456Z", val)
+			assert.Equal(t, "09:10:11.123456", val)
```

Comment:
This is now consistent with how we emit `TIME` values which is to not include a TZ locale.